client/a8c-for-agencies/components/content-sidebar/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Main from 'calypso/components/main';
+import SidebarNavigation from 'calypso/jetpack-cloud/sections/partner-portal/sidebar-navigation';
+
+import './style.scss';
+
+type Props = {
+	mainContent: React.ReactNode;
+	rightSidebar: React.ReactNode;
+};
+
+const ContentSidebar = ( { mainContent, rightSidebar }: Props ) => {
+	return (
+		<>
+			<SidebarNavigation />
```

Comment:
I'm wondering if we actually need it here.

Reply:
Yes, correct. We already have this as a part of the Layout.